src/index.html

```diff
@@ -1,13 +1,293 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link rel="shortcut icon" href="images/icons/favicon.ico">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      type="image/vnd.microsoft.icon"
+      href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo">
+              <img src="images/logo-2x.png" alt="Bose logo">
+            </a>
+            <div class="top-bar__icons">
+              <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
```

Comment:
Need to fix format phone only in href everywhere
```suggestion
              <a href="tel:+123455555555" class="icon icon--phone"></a>
```